appserver/payara-appserver-modules/microprofile/openapi/src/main/java/fish/payara/microprofile/openapi/impl/OpenApiService.java

```diff
@@ -262,12 +257,12 @@ private static String getContextRoot(ApplicationInfo appInfo) {
         private final OpenApiConfiguration appConfig;
         private volatile OpenAPI document;
 
-        private OpenApiMapping(ApplicationInfo appInfo) {
+        OpenApiMapping(ApplicationInfo appInfo) {
```

Comment:
Is there any reason for this?

Reply:
Short: Synthetic methods. Long explanation given in another comment.